index.html

```diff
@@ -464,6 +468,41 @@ <h3>CSS</h3>
                 </ul>
             </div>
         </section>
+        <section class="contact">
+            <div class="content">
+                <div class="txt-holder">
+                    <h1 class="primary-txt">
+                        Contact me
+                    </h1>
+                    <p class="detail-txt">
+                        If you have an application you
+                        are interested in developing, a
+                        feature that you need built or a
+                        project that needs coding. I’d
+                        love to help with it
+                    </p>
+                </div>
+
+                <div class="contact-form">
+                    <form action="#" method="post">
+                        <ul>
+                            <li>
+                                <input type="text" placeholder="Write your name here"  name="user_name" id="name">
+                            </li>
+                            <li>
+                                <input type="email" placeholder="Write your email here" name="user_email" id="mail">
+                            </li>
+                            <li>
+                                <textarea placeholder="Write your message here" name="user_message" id="msg" ></textarea>
```

Comment:
- [ ] I'm suggesting you also please add the `required` attribute for the message field also. In addition, the message input should only accept 500 characters. In that case, please set a `maxlength` attribute with the value 500 assigned to it also.